.drone.yml

```diff
@@ -717,7 +717,7 @@ steps:
 
   - name: publish
     pull: always
-    image: plugins/docker:linux-amd64
+    image: techknowlogick/drone-docker:linux-amd64
```

Comment:
Can you update to use `:latest` tag? It is multi-arch so no need to target specific arch.

Reply:
hehe sure